### ISSUE TITLE
Introduction to the use of datasets for the provided pre-trained models

### ISSUE THREAD

**Issue by XinMing0411** [LAION-AI/CLAP#107]
Hi all！
What are the datasets used in the pre-trained model provided in the Google link?
Were [630k-audioset-best.pt](https://huggingface.co/lukewys/laion_clap/blob/main/630k-audioset-best.pt) and [630k-audioset-fusion-best.pt](https://huggingface.co/lukewys/laion_clap/blob/main/630k-audioset-fusion-best.pt) pre-trained models trained only on the Audioset and LAION-Audio-630k datasets? Do their training data include the Clotho and AudioCaps datasets?

**Comment by dilipupf** [comment]
I have my own custom evaluation dataset similar to clotho evaluation dataset for which i just want to do inference on the pretrained checkpoint and compute the recall metric. how can i do it? i tried reading through the documentation it's a bit confusing to me. Appreciate your help!
